apps/meteor/server/startup/migrations/v313.ts

```diff
@@ -0,0 +1,118 @@
+import type { ILoginServiceConfiguration, OAuthConfiguration } from '@rocket.chat/core-typings';
+import { Settings, LoginServiceConfiguration } from '@rocket.chat/models';
+
+import { settings } from '../../../app/settings/server';
```

Comment:
We can't use the cached setting in migrations, please use the model instead.

Reply:
Done!